File: c language/question 18.c
// 18. Print all elements that are at even indexes in the array arr = [10, 20, 30, 40, 50].
#include <stdio.h>
int main(){
    int arr[]={10,20,30,40,50};
    int len=sizeof(arr)/sizeof(arr[0]);
    for(int i=1;i<len;i++){
        if(i%2==0){
            printf("%d",arr[i]);
        }
    }
    return 0;
}

File: Java script/question 18.js
// 18. Print all elements that are at even indexes in the array arr = [10, 20, 30, 40, 50].
var arr = [10, 20, 30, 40, 50]
var len=arr.length
for(i=1;i<len;i++){
    if(i%2==0){
        console.log(arr[i])
    }
}